src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,39 @@
+package duke.command;
+
+import duke.exception.OutOfRangeException;
+//
+import duke.storage.Storage;
+//
+import duke.task.Task;
+import duke.task.TaskList;
+//
+import duke.ui.Ui;
+
+public class DeleteCommand extends Command {
+    private String response;
+
+    public DeleteCommand(String response) {
+        this.response = response;
+    }
+
+    /**
+     * Shows the task just be deleted.
+     *
+     * @param tasks The list of tasks.
+     * @param ui The user interaction instance.
+     * @param storage The instance to store data.
+     */
+    @Override
+    public String execute(TaskList tasks, Ui ui, Storage storage) throws OutOfRangeException {
+        //check with the special response "delete X", where X is index of deleted item.
+        int curr = Integer.parseInt(response.substring(7));
```

Comment:
Perhaps it is better to use a named constant like ```LEN_OF_DELETE``` to avoid magic numbers.
```suggestion
        int curr = Integer.parseInt(response.substring(LEN_OF_DELETE));
```